packages/app-admin-ui/client/pages/List/Management.js

```diff
@@ -60,7 +60,7 @@ const ListManage = ({ list, pageSize, totalItems, selectedItems, onDeleteMany, o
           </IconButton>
         )}
 
-        {list.access.update && (
+        {list.access.delete && (
```

Comment:
this one seems to be bug earlier where you have delete access but no update access then the delete button for multiple itmes not available in list view.